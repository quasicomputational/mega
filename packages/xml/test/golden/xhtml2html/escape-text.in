<html xmlns="http://www.w3.org/1999/xhtml"><head /><body>&amp; &quot; &apos; &#xA0; &lt; &gt;</body></html>
