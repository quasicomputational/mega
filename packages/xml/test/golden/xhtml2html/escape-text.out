<!DOCTYPE html><html><head></head><body>&amp; &quot; ' &nbsp; &lt; &gt;</body></html>